### PR TITLE
New VMs: include netavark in prior-fedora

### PR DESCRIPTION
**Pull request by edsantiago**
CNI is deprecated, and will no longer be tested in CI (Podman
PR 21410).

We've been force-removing netavark from prior-fedora. Remove
this special case so now all fedorae have netavark.

Signed-off-by: Ed Santiago <santiago@redhat.com>
```release-note
None
```

### CODE REVIEW

**Comment by edsantiago** [comment]
Compared to #324

|  |*debian*|*prior-fedora*|*fedora*|*fedora-aws*|*rawhide*|
|----|----|----|----|----|----|
|*base*| 13 | 38-1.6 | 39-1.5 | - | 40-0 |
|*aardvark-dns*| 1.4.0-5 | 1.9.0-1 | - &lt;- 1.9.0-1 | 1.9.0-1 | - &lt;- 1.9.0-1 |
|*buildah*| 1.33.3+ds1-2 | 1.33.2-1 | 1.34.0-1 | 1.33.2-1 | 1.34.0-1 |
|*conmon*| 2.1.6+ds1-1 | 2:2.1.8-2 | 2:2.1.8-2 | 2:2.1.8-2 | 2:2.1.8-2 |
|*container-selinux*| - | 2:2.227.0-1 | 2:2.229.0-1 &lt;- 2:2.228.1-1 | 2:2.228.1-1 | 2:2.229.0-1 |
|*containers-common*| - | 4:1-89 | 4:1-95 | 4:1-95 | 5:0.57.1-1 |
|*criu*| 3.17.1-3 | 3.18-1 | 3.19-2 | 3.19-2 | 3.19-2 |
|*crun*| 1.14-1 | 1.12-1 | 1.14-1 | 1.13-1 | 1.14-1 |
|*golang*| 2:1.21~2 | 1.20.13-1 | 1.21.6-1 | 1.21.6-1 | 1.22~rc2-1 &lt;- 1.21.6-1 |
|*netavark*| 1.4.0-4 | 1.9.0-1 | 1.10.1-1 &lt;- 1.9.0-1 | 1.9.0-1 | 1.10.1-1 &lt;- 1.9.0-1 |
|*passt*| 2023-12-30 | 2023-12-30 | - | 2023-12-30 | 2023-12-30 |
|*podman*| 4.9.0+ds1-2 | 5:4.8.3-1 | 5:4.9.0-1 | 5:4.8.3-1 | 5:4.9.0-1 |
|*runc*| 1.1.10+ds1-1 | 2:1.1.8-1 | 2:1.1.8-1 | 2:1.1.8-1 | 2:1.1.9-1 |
|*skopeo*| 1.13.3+ds1-2 | 1:1.14.1-1 &lt;- 1:1.14.0-1 | 1:1.14.1-1 | 1:1.14.0-1 | 1:1.14.1-1 |
|*slirp4netns*| 1.2.1-1 | 1.2.2-1 | 1.2.2-1 | 1.2.2-1 | 1.2.2-1 |
|*systemd-container*| 255.3-2 &lt;- 255.3-1 | 253.15-2 &lt;- 253.14-1 | 254.8-2 | 254.8-2 | 255.3-1 &lt;- 255.2-2 |
|*tar*| - | - | - | - | 2:1.35-2 |

**Comment by edsantiago** [comment]
I guess we'll find out soon enough, but the `aardvark` line bothers me. It looks like no aardvark got installed for f39 and rawhide?

**Comment by ashley-cui** [comment]
Looks like aardvark needs to be included, trips in https://github.com/containers/podman/pull/21366. Weird because it doesn't trip in our current f39, so it has to be installed there right now?

**Comment by edsantiago** [comment]
See group email. This is (I'm 99% sure) fallout from this weekend's netavark disaster. I just don't understand why that happened.

**Comment by cevich** [comment]
Whew! Okay I think they're all closed and linked back to this PR.  I can't find a Renovate PR for the podman repo.  Was the v20240125 image put in place manually there?

I hope you don't feel bad Ed, this could have happened to anybody.

**Comment by edsantiago** [comment]
https://github.com/containers/podman/pull/21366 _was_ my PR for the 0125 image, but since it was stalled I decided to repush it with the new 0129 images from today. Then kablooey.

**Comment by cevich** [comment]
> Then kablooey.

Gotca, so it sounds like there **is** a risk Renovate could still open a 0125 PR for podman.  I've seen it "sleep" on update PRs before, and I also don't have the (deliberately) set update schedules all memorized :thinking: 

I'm now completely and thoroughly trigger shy when it comes to sending any warning messages to the group.  On the off-chance I happen to use some unapproved word.  If a "podman-update PR: do not merge" warning mail is urgent and important, somebody else can write it.

File: cache_images/fedora_packaging.sh [hunk 1]
```diff
@@ -37,7 +37,11 @@ msg "Updating/Installing repos and packages for $OS_REL_VER"
 
 bigto ooe.sh $SUDO dnf update -y
 
+# netavark crisis of 2024-01-27 weekend
+timebomb 20240210 "Please remove aardvark-dns from INSTALL_PACKAGES; netavark should Require it"
+
```
Comment (cevich) [inline cache_images/fedora_packaging.sh:42]: Thanks for adding this.

**Review by cevich — APPROVED** [review]
LGTM

**Comment by cevich** [comment]
I found Renovate's podman update task queued due to rate-limiting, [in the dependency dashboard](https://github.com/containers/podman/issues/18109).  So indeed it was on a hair-trigger as I feared.  Anyhoo, I'm glad this didn't happen in the middle of the night.  I tripped it manually just now, and when the PR shows up I'll close it to prevent any accidents.

**Comment by edsantiago** [comment]
Comparing against last wee's build

|  |*debian*|*prior-fedora*|*fedora*|*fedora-aws*|*rawhide*|
|----|----|----|----|----|----|
|*base*| 13 | 38-1.6 | 39-1.5 | - | 40-0 |
|*aardvark-dns*| 1.4.0-5 | 1.9.0-1 | 1.10.0-1 &lt;- 1.9.0-1 | 1.9.0-1 | 1.9.0-1 |
|*buildah*| 1.33.3+ds1-2 | 1.33.2-1 | 1.34.0-1 | 1.33.2-1 | 1.34.0-1 |
|*conmon*| 2.1.6+ds1-1 | 2:2.1.8-2 | 2:2.1.8-2 | 2:2.1.8-2 | 2:2.1.8-2 |
|*container-selinux*| - | 2:2.227.0-1 | 2:2.229.0-1 &lt;- 2:2.228.1-1 | 2:2.228.1-1 | 2:2.229.0-1 |
|*containers-common*| - | 4:1-89 | 4:1-95 | 4:1-95 | 5:0.57.1-1 |
|*criu*| 3.17.1-3 | 3.18-1 | 3.19-2 | 3.19-2 | 3.19-2 |
|*crun*| 1.14-1 | 1.12-1 | 1.14-1 | 1.14-1 &lt;- 1.13-1 | 1.14-1 |
|*golang*| 2:1.21~2 | 1.20.13-1 | 1.21.6-1 | 1.21.6-1 | 1.22~rc2-1 &lt;- 1.21.6-1 |
|*netavark*| 1.4.0-4 | 1.9.0-1 | 1.10.1-5 &lt;- 1.9.0-1 | 1.9.0-1 | 1.10.1-1 &lt;- 1.9.0-1 |
|*passt*| 2023-12-30 | 2023-12-30 | - | 2023-12-30 | 2023-12-30 |
|*podman*| 4.9.0+ds1-2 | 5:4.8.3-1 | 5:4.9.0-1 | 5:4.9.0-1 &lt;- 5:4.8.3-1 | 5:4.9.0-1 |
|*runc*| 1.1.10+ds1-1 | 2:1.1.8-1 | 2:1.1.8-1 | 2:1.1.8-1 | 2:1.1.9-1 |
|*skopeo*| 1.13.3+ds1-2 | 1:1.14.1-1 &lt;- 1:1.14.0-1 | 1:1.14.1-1 | 1:1.14.0-1 | 1:1.14.1-1 |
|*slirp4netns*| 1.2.1-1 | 1.2.2-1 | 1.2.2-1 | 1.2.2-1 | 1.2.2-1 |
|*systemd-container*| 255.3-2 &lt;- 255.3-1 | 253.15-2 &lt;- 253.14-1 | 254.8-2 | 254.8-2 | 255.3-1 &lt;- 255.2-2 |
|*tar*| - | - | - | - | 2:1.35-2 |

**Comment by edsantiago** [comment]
Thanks Chris. It's not a big deal if it had merged, but it wouldn't have merged anyway because CI would've failed.

**Comment by cevich** [comment]
> it wouldn't have merged anyway because CI would've failed.

Oh whew!  Thanks for the extra perspective.  It appeared like a dooms-day thing to me.  Glad it's not.

**Comment by edsantiago** [comment]
Netavark-1.9 is gone from rawhide; I cannot install it.

Netavark-1.10 is [broken](https://github.com/containers/netavark/pull/911). A [fix is in progress](https://github.com/containers/netavark/pull/912) but then it'll have to get built and bodhi'ed etc etc.

**Comment by edsantiago** [comment]
Sigh. Still broken netavark. No point in testing this in podman.

- 20240201t143038z-f39f38d13
- 20240125t184057z-f39f38d13 &#x21D1; (built in #324)

|  |*debian*|*prior-fedora*|*fedora*|*fedora-aws*|*rawhide*|
|----|----|----|----|----|----|
|*base*         | 13              | 38-1.6          | 39-1.5          | ?               | 40-0            |
|               |                |                |                |                |                |
|*aardvark-dns* | 1.4.0-5         | 1.9.0-1         | 1.10.0-1        | 1.9.0-1         | 1.10.0-1        |
|               |                |                | 1.9.0-1 &#x21D1; |                | 1.9.0-1 &#x21D1; |
|*buildah*      | 1.33.3+ds1-2    | 1.33.2-1        | 1.34.0-1        | 1.34.0-1        | 1.34.0-3        |
|               |                |                |                | 1.33.2-1 &#x21D1; | 1.34.0-1 &#x21D1; |
|*conmon*       | 2.1.6+ds1-1     | 2:2.1.8-2       | 2:2.1.8-2       | 2:2.1.8-2       | 2:2.1.8-4       |
|               |                |                |                |                | 2:2.1.8-2 &#x21D1; |
|*container-selinux*| ?               | 2:2.227.0-1     | 2:2.229.0-1     | 2:2.229.0-1     | 2:2.229.0-2     |
|               |                |                | 2:2.228.1-1 &#x21D1; | 2:2.228.1-1 &#x21D1; | 2:2.229.0-1 &#x21D1; |
|*containers-common*| ?               | 4:1-89          | 4:1-95          | 4:1-95          | 5:0.57.1-7      |
|               |                |                |                |                | 5:0.57.1-1 &#x21D1; |
|*criu*         | 3.17.1-3        | 3.18-1          | 3.19-2          | 3.19-2          | 3.19-4          |
|               |                |                |                |                | 3.19-2 &#x21D1; |
|*crun*         | 1.14-1          | 1.12-1          | 1.14-1          | 1.14-1          | 1.14-1          |
|               |                |                |                | 1.13-1 &#x21D1; |                |
|*golang*       | 2:1.21~2        | 1.20.13-1       | 1.21.6-1        | 1.21.6-1        | 1.22~rc2-1      |
|               |                |                |                |                | 1.21.6-1 &#x21D1; |
|*netavark*     | 1.4.0-4         | 1.9.0-1         | 1.10.1-5        | 1.10.1-5        | 1.10.1-5        |
|               |                |                | 1.9.0-1 &#x21D1; | 1.9.0-1 &#x21D1; | 1.9.0-1 &#x21D1; |
|*passt*        | 2023-12-30      | 2023-12-30      | ?               | 2023-12-30      | 2023-12-30      |
|               |                |                |                |                |                |
|*podman*       | 4.9.0+ds1-2     | 5:4.8.3-1       | 5:4.9.0-1       | 5:4.9.0-1       | 5:4.9.0-1       |
|               |                |                |                | 5:4.8.3-1 &#x21D1; |                |
|*runc*         | 1.1.10+ds1-1    | 2:1.1.8-1       | 2:1.1.8-1       | 2:1.1.8-1       | 2:1.1.9-2       |
|               |                |                |                |                | 2:1.1.9-1 &#x21D1; |
|*skopeo*       | 1.13.3+ds1-2    | 1:1.14.1-1      | 1:1.14.1-1      | 1:1.14.0-1      | 1:1.14.1-2      |
|               |                | 1:1.14.0-1 &#x21D1; |                |                | 1:1.14.1-1 &#x21D1; |
|*slirp4netns*  | 1.2.1-1         | 1.2.2-1         | 1.2.2-1         | 1.2.2-1         | 1.2.2-2         |
|               |                |                |                |                | 1.2.2-1 &#x21D1; |
|*systemd-container*| 255.3-2         | 253.15-2        | 254.8-2         | 254.8-2         | 255.3-1         |
|               | 255.3-1 &#x21D1; | 253.14-1 &#x21D1; |                |                | 255.2-2 &#x21D1; |
|*tar*          | ?               | ?               | ?               | ?               | 2:1.35-3        |
|               |                |                |                |                | 2:1.35-2 &#x21D1; |

**Comment by edsantiago** [comment]
@cevich PTAL when convenient, and merge if acceptable. TL;DR our lack of VMs is causing cascading blocks of important CI changes. Podman tests have passed: https://github.com/containers/podman/pull/21366

I will open a new PR next week to try to bring in a working netavark.

**Comment by cevich** [comment]
Thanks Ed.  Do you have an opinion on if this version should be tagged?  Or, should we maybe wait a few days at least maybe?

**Comment by edsantiago** [comment]
No, absolutely do not tag. Netavark is broken. I will be spinning new VMs again next week. Thanks.

**Pull request merged**